build.gradle.kts

```diff
@@ -87,13 +89,29 @@ tasks.withType<Jar>().configureEach {
 
 application.mainClassName = "org.gradle.profiler.Main"
 
+node {
+    download = true
+}
+
+val generateHtmlReportJavaScript = tasks.register<NpxTask>("generateHtmlReportJavaScript") {
```

Comment:
Shall we move this task to a task class in `buildSrc` and maybe add a convention plugin in build src which does set up the Javascript parts?

Reply:
#259 adds `buildSrc`. Let's not conflict with them, and do this afterwards.

Reply:
👍